### PR TITLE
[Js] fetch type drink as category selected

### LINKED ISSUE #47: [Js] fetch type drink as category selected (typeDrink page)

### PR DESCRIPTION
**Pull request by muhannaAtalla20**
Relates #47

### CODE REVIEW

**Review by MohammedDahalan — APPROVED** [review]

File: otherPages/type/typeDrink.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
     <!-- Start Search Section -->
     <section class = "meal-search-box">
             <div class="header-search">
-                <h2>Ordiary Drink Category</h2>
+                <h2></h2>
```
Comment (yaqootturman) [inline otherPages/type/typeDrink.html]: no need for it

File: otherPages/type/typeFood.html [hunk 2]
```diff
@@ -30,10 +30,10 @@
     <!-- Start Search Section -->
     <section class = "meal-search-box">
             <div class="header-search">
-                <h2>Seafood Category</h2>
+                <h2></h2>
```
Comment (yaqootturman) [inline otherPages/type/typeFood.html]: same

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: otherPages/type/typeDrink.html (hunk 1 above)
Comment (muhannaAtalla20) [reply otherPages/type/typeDrink.html]: done

File: otherPages/type/typeFood.html (hunk 2 above)
Comment (muhannaAtalla20) [reply otherPages/type/typeFood.html]: done

**Review by yaqootturman — APPROVED** [review]

**Review by IsraaHamdi — APPROVED** [review]

**Pull request merged**